Makefile

```diff
@@ -89,7 +89,7 @@ endif
 
 .PHONY: build
 build: generate fmt vet
-	build/build.sh ${OUTPUT_DIR} ${GO_MOD_FLAGS}
+	build/build.sh ${OUTPUT_DIR} $(VERSION) ${GO_MOD_FLAGS}
```

Comment:
Adding `${VERSION}` is very confusing given it is going away in the next commit. Not sure what this achieves. This seems outside of using the new bundle.